README.md

```diff
@@ -31,7 +27,7 @@ netanalysis is not currently in PyPI.  Instead you can install straight from our
 This uses the Google Transparency Report internal API. You must agree to [Google APIs Terms of Service](https://developers.google.com/terms/).
 
 ```
-.venv/bin/python -m netanalysis.traffic.data.fetch_google_traffic --output_dir=traffic_data/
+python -m netanalysis.traffic.data.fetch_google_traffic --output_dir=traffic_data/
```

Comment:
It won't let me comment there but above on line 14 where we give the pip installation instructions let's also document how to depend on netanalysis in a Pipfile.

Reply:
Done